pulp_2to3_migration/app/plugin/rpm/migrator.py

```diff
@@ -121,11 +168,114 @@ def pipeline_stages(self):
             RemoteArtifactSaver(),
             InterrelateContent(),
             RelatePulp2to3Content(),
+
         ]
 
         return pipeline
 
 
+class RpmContentMigrationFirstStage(ContentMigrationFirstStage):
+    """
+    The first stage of a Rpm content migration pipeline.
+    """
+
+    async def migrate_to_pulp3(self, batch, migrator, pb=None):
```

Comment:
@ipanova , after the refactor, do we still need our own `migrate_to_pulp3` for rpm plugin?